src/pages/wallet/select.currency.js

```diff
@@ -43,7 +44,7 @@ class WalletSelectCurrency extends Component {
       const { navigation } = this.props;
       this.state = {
         isLoading: false,
-        isDisabledSwitch: false,
+        isCanSubmit: true,
```

Comment:
The same as "is", "can" can used solely as the start of a variable, so "canSubmit" is fine.